_posts/2018-11-21-f18_01.md

```diff
@@ -0,0 +1,46 @@
+---
+layout: post
+title: "Ecung-Ibbetson House"
+date: 2018-03-12
```

Comment:
The date here needs to be a date during this quarter. Please edit, then resubmit your pull request. Thanks!

Reply:
I changed the date.